actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -904,6 +904,16 @@ def permitted_scalar?(value)
         PERMITTED_SCALAR_TYPES.any? { |type| value.is_a?(type) }
       end
 
+      # Adds existing keys to the params if their values are scalar.
+      #
+      # For example:
+      #
+      #   puts self.keys #=> ["zipcode(90210i)"]
+      #   params = {}
+      #
+      #   permitted_scalar_filter(params, "zipcode")
+      #
+      #   puts params.keys # => ["zipcode"]
       def permitted_scalar_filter(params, key)
```

Comment:
Do we intend to expose the private method to the api doc?

Reply:
I don't think that was the intention here. Even that the method is private it benefits from documentation to help us to understand what it does without the need to look the code.